### PR TITLE
Updated the India LGD districts data

### PR DESCRIPTION
**Pull request by thejeshgn**
- Fixes https://github.com/datacommonsorg/data/issues/740
- Updated the testcases to reflect the wikidata changes listed below
- Check - [MCF Browser](https://datacommonsorg.github.io/tools/mcf-browser/#&file=https://raw.githubusercontent.com/datacommonsorg/data/de9618e038cb003ec37eb4c94b0a091a93176c8a/scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf&file=https://raw.githubusercontent.com/datacommonsorg/data/de9618e038cb003ec37eb4c94b0a091a93176c8a/scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.csv)

### WikiData Changes

Warangal district http://www.wikidata.org/entity/Q28169759
Now  Redirects to  https://www.wikidata.org/wiki/Q15399
- - -
New Delhi district http://www.wikidata.org/entity/Q110067863
Now redirects to https://www.wikidata.org/wiki/Q8560886
- - -
Malerkotla https://www.wikidata.org/wiki/Q1470987
instead of converting, they have created a new entity
https://www.wikidata.org/wiki/Q107016021
- - -

### CODE REVIEW

File: scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
 Node: E:LocalGovernmentDirectory_Districts->E1
 typeOf: dcs:AdministrativeArea2
+dcid: C:LocalGovernmentDirectory_Districts->DistrictDCID 
 wikidataId: C:LocalGovernmentDirectory_Districts->WikiDataId
 name: C:LocalGovernmentDirectory_Districts->LGDDistrictNameTitleCase
 lgdCode: C:LocalGovernmentDirectory_Districts->LGDDistrictCode
 indianCensusAreaCode2011: C:LocalGovernmentDirectory_Districts->LGDCensus2011Code
 indianCensusAreaCode2001:  C:LocalGovernmentDirectory_Districts->LGDCensus2001Code
-containedInPlace: E:LocalGovernmentDirectory_Districts->E0
+containedInPlace: E:LocalGovernmentDirectory_Districts->StateDCID
```
Comment (ajaits) [inline scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf:9]: Please change to column reference form node referes with `C:` prefix:
`containedInPlace: C:LocalGovernmentDirectory_Districts->StateDCID`

File: scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.csv [hunk 2]
```diff
@@ -1,740 +1,758 @@
...
+614,kiphire,13,nagaland,13.02,269,kiphire,http://www.wikidata.org/entity/Q2597908,kiphire,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,269,Q2597908,Kiphire,Kiphire,wikidataId/Q1599,wikidataId/Q2597908
+615,longleng,13,nagaland,13.02,268,longleng,http://www.wikidata.org/entity/Q1426783,longleng,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,268,Q1426783,Longleng,Longleng,wikidataId/Q1599,wikidataId/Q1426783
+736,noklak,13,nagaland,,,noklak,http://www.wikidata.org/entity/Q48731903,noklak,"district in India, Nagaland",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,,Q48731903,Noklak,Noklak,wikidataId/Q1599,wikidataId/Q48731903
+757,tseminyu,13,nagaland,,,tseminyu,http://www.wikidata.org/entity/Q110223836,tseminyu,"district of Nagaland, India",http://www.wikidata.org/wntity/Q1599,nagaland,Indian state,,Q110223836,Tseminyu,Tseminyu,wikidataId/http://www.wikidata.org/wntity/Q1599,wikidataId/Q110223836
```
Comment (ajaits) [inline scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.csv:293]: pls remove url from the StatDCID column:
`wikidataId/http://www.wikidata.org/wntity/Q1599` -> `wikidataId/Q1599`

Comment (thejeshgn) [reply scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.csv]: Fixed.

File: scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf (hunk 1 above)
Comment (thejeshgn) [reply scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf]: Fixed

**Review by ajaits — APPROVED** [review]

**Pull request merged**